packages/flutter/test/services/text_input_test.dart

```diff
@@ -140,12 +140,28 @@ void main() {
       const TextInputType decimal = TextInputType.numberWithOptions(decimal: true);
       const TextInputType signedDecimal =
         TextInputType.numberWithOptions(signed: true, decimal: true);
+      const TextInputType multiline = TextInputType.multiline;
+      const TextInputType phone = TextInputType.phone;
+      const TextInputType datetime = TextInputType.datetime;
+      const TextInputType emailAddress = TextInputType.emailAddress;
+      const TextInputType url = TextInputType.url;
+      const TextInputType visiblePassword = TextInputType.visiblePassword;
+      const TextInputType name = TextInputType.name;
+      const TextInputType streetAddress = TextInputType.streetAddress;
 
```

Comment:
Instead of testing that each value has an index equal to the index you used to retrieve it (which is trivial). Maybe just test the expected index for each value?